WordPress/src/main/java/org/wordpress/android/datasets/PeopleTable.java

```diff
@@ -26,6 +25,7 @@ private static SQLiteDatabase getWritableDb() {
     public static void createTables(SQLiteDatabase db) {
         db.execSQL("CREATE TABLE " + PEOPLE_TABLE + " ("
                 + "person_id               INTEGER DEFAULT 0,"
+                + "site_id                 TEXT,"
```

Comment:
Ok, `blog_id` better matches other field/var names. BTW, we won't use the word blog at all in wpstores, so that renaming will come naturally when we integrate wpstores into wpandroid.